web/api/v1/api.go

```diff
@@ -822,22 +822,39 @@ func (api *API) metricMetadata(r *http.Request) apiFuncResult {
 		}
 	}
 
+	metric := r.FormValue("metric")
+
 	for _, tt := range api.targetRetriever.TargetsActive() {
 		for _, t := range tt {
-			for _, mm := range t.MetadataList() {
-				m := metadata{Type: mm.Type, Help: mm.Help, Unit: mm.Unit}
-				ms, ok := metrics[mm.Metric]
+
+			if metric == "" {
```

Comment:
Does this work @brian-brazil? Will update the docs and PR description if agreed.

Was tempted to DRY up the appending of new metadata but refrained from that for now.

Reply:
Looks okay, I didn't recall that we had the t.Metadata.

Reply:
We always did.

Reply:
I can't remember every PR to add a new feature that I reviewed :)

Reply:
sorry if I implied that with my comment was definitely not my intention, I came from an angle of a confirmation.

Reply:
Oh no, I like everyone else forget things sometimes and I was joking about that.